lib/Jsrt/Core/JsrtCore.cpp

```diff
@@ -227,3 +227,20 @@ JsGetModuleHostInfo(
     });
     return errorCode;
 }
+
+CHAKRA_API JsGetModuleNamespace(_In_ JsModuleRecord requestModule, _Outptr_result_maybenull_ JsValueRef *moduleNamespace)
+{
+    PARAM_NOT_NULL(moduleNamespace);
+    *moduleNamespace = nullptr;
+    if (!Js::SourceTextModuleRecord::Is(requestModule))
+    {
+        return JsErrorInvalidArgument;
+    }
+    Js::SourceTextModuleRecord* moduleRecord = Js::SourceTextModuleRecord::FromHost(requestModule);
+    if (!moduleRecord->WasEvaluated())
+    {
+        return JsErrorModuleNotEvaluated;
+    }
+    *moduleNamespace = (JsValueRef) moduleRecord->GetNamespace();
```

Comment:
>(JsValueRef) [](start = 23, length = 12)

I prefer explicit casting operators like static_cast or reinterpret_cast because their behavior is much clearer than C-style casts.